README.md

```diff
@@ -24,9 +19,8 @@ npm install && npm run dev
 
 ## Deploy Preview
 
-This repo is configured to automatically create a preview environment
-on Vercel when a PR is opened. The Vercel bot will leave a comment with a link to the preview on your PR.
+This repo is configured to automatically create a preview environment on Vercel when a PR is opened. After the deployment is approved, the Vercel bot will leave a comment with a link to the preview on your PR.
 
 ## Deploying to Production
 
-This repo is configured to automatically update production (`https://docs.chain.link`) when the `main` branch is updated.
+This repo is configured to automatically update the production (`https://docs.chain.link`) site when commits are pushed to the `main` branch.
```

Comment:
Your changes are looks good (sweet and simple). in addition I wanted add something like this 👇, let me know what's your views on that

## Contributing

Contributions are what make the open source community such an amazing place to learn, inspire, and create. Any contributions you make are **greatly appreciated**.

If you have a suggestion that would make this better, please fork the repo and create a pull request. You can also simply open an issue with the tag "enhancement".
Don't forget to give the project a star⭐! Thanks again!

1. Fork the Project & Raise an issue 
2. Create your Branch 
3. Commit your Changes 
4. Push to the Branch 
5. Open a Pull Request

Reply:
Yes that would be helpful in the [Contributing](https://github.com/smartcontractkit/documentation/blob/main/CONTRIBUTING.md) guide. You can add that as part of this PR or make a separate PR. Whatever works best for you. Thank you!

Reply:
Sure 👍